docs/intro/ff-ui/canvas.md

```diff
@@ -0,0 +1,144 @@
+---
+slug: canvas
+title: Canvas
+tags: []
+description: Dive into the versatile Canvas in FlutterFlow, where you can effortlessly design and preview your app’s interface.
+sidebar_position: 2
+---
+
+# Canvas 
+The Canvas shows the screen of the device, which could be mobile, iPad, web, or desktop. It allows you to add widgets via drag-and-drop. You can select, move, and position widgets anywhere on the Canvas.
+
+It allows zoom level adjustments and previews in light or dark mode. It also includes features for multi-language preview, adding App Bar and Nav Bar, simulating larger font visualization, and more.
+
+![canvas area](imgs/canvas-new.avif)
```

Comment:
Thank you for redoing the graphic again! Love it!

Also lets remove the numbering from the headers since the image doesnt have it anymore.

Reply:
Done!